### PR TITLE
Add simple checks for MicroOS system role

### PR DESCRIPTION
**Pull request by kravciak**
Local runs:
 - http://dhcp91.suse.cz/tests/1375 - Kubic
 - http://dhcp91.suse.cz/tests/1378 - CaaSP

@sysrich please provide some input about what to check for (MicroOS vs Plain)

### CODE REVIEW

**Comment by sysrich** [comment]
That looks pretty good to me. 

I can't think of anything from https://en.opensuse.org/Kubic:MicroOS that would help more

File: tests/caasp/one_line_checks.pm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright © 2017 SUSE LLC
+# Copyright © 2018 SUSE LLC
```
Comment (asmorodskyi) [inline tests/caasp/one_line_checks.pm]: usually we change to 2017-2018

**Comment by okurz** [comment]
update copyright year, then LGTM

File: tests/caasp/one_line_checks.pm (hunk 1 above)
Comment (kravciak) [reply tests/caasp/one_line_checks.pm]: Fixed, thanks

**Pull request merged**